pwa-installer/README.md

```diff
@@ -0,0 +1,36 @@
+# PWA installer
+
+➡️ **[Open the demo](https://microsoftedge.github.io/Demos/pwa-installer/)** ⬅️
+
+**PWA installer** is a demo app for a landing page of a collection of applications.  This demo application showcases the Web Install API and the CSS Masonry layout.
```

Comment:
CSS Masonry is used more as a progressive enhancement here. The Web Install API is the real point of this demo. 

```suggestion
**PWA installer** is a demo app for a landing page that lets you install PWAs from a collection of applications.  This demo application showcases the Web Install API.
```